dds/DCPS/RTPS/Sedp.h

```diff
@@ -348,62 +349,71 @@ class Sedp : public DCPS::EndpointManager<ParticipantData_t> {
     void remove_associations(const DCPS::ReaderIdSeq&, bool) {}
     void retrieve_inline_qos_data(InlineQosData&) const {}
 
+    const DCPS::SequenceNumber& get_seq() const
+    {
+      return seq_;
+    }
+
+  protected:
     void send_sample(const ACE_Message_Block& data,
-                     size_t size,
-                     const DCPS::RepoId& reader,
-                     DCPS::SequenceNumber& sequence,
-                     bool historic = false);
+      size_t size,
```

Comment:
doesn't look like this needs to change in this PR (multiple occurrences)

Reply:
Fixed